### ISSUE TITLE
Deployment error with Heroku

### ISSUE THREAD

**Issue by rajeefmk** [akhater/Lupin#19]
Hi, I was able to successfully test Lupin locally using Docker container but when deployed to Heroku, it came up with the following error: 

```
2021-04-04T18:18:53.519062+00:00 heroku[web.1]: Starting process with command `python -u main.py`
2021-04-04T18:18:56.907306+00:00 app[web.1]: Traceback (most recent call last):
2021-04-04T18:18:56.907339+00:00 app[web.1]:   File "/app/main.py", line 7, in <module>
2021-04-04T18:18:56.907446+00:00 app[web.1]:     import flashcards
2021-04-04T18:18:56.907447+00:00 app[web.1]:   File "/app/flashcards.py", line 10, in <module>
2021-04-04T18:18:56.907551+00:00 app[web.1]:     from config import getflashcardsTag
2021-04-04T18:18:56.907552+00:00 app[web.1]:   File "/app/config.py", line 14, in <module>
2021-04-04T18:18:56.907650+00:00 app[web.1]:     BotToken = config.get('Bot','BotToken')
2021-04-04T18:18:56.907651+00:00 app[web.1]:   File "/usr/local/lib/python3.9/configparser.py", line 781, in get
2021-04-04T18:18:56.907975+00:00 app[web.1]:     d = self._unify_values(section, vars)
2021-04-04T18:18:56.907975+00:00 app[web.1]:   File "/usr/local/lib/python3.9/configparser.py", line 1149, in _unify_values
2021-04-04T18:18:56.908355+00:00 app[web.1]:     raise NoSectionError(section) from None
```

I'm assuming this is not a lupin related error but wondering if anyone had success deploying to Heroku. 

I used the [heroku.yml](https://devcenter.heroku.com/articles/build-docker-images-heroku-yml#getting-started) to deploy the Dockerfile.

**Comment by digitalknk** [comment]
You may actually benefit from running it via their python buildpack instead of the container image.

I'll have to see if it has something to do with the container image or just how heroku starts it.

**Comment by rajeefmk** [comment]
Thanks! Do you have any other alt platform suggestion for quick deploying? Heroku was the first one that came to mind but I'm happy to try out others.

**Comment by akhater** [comment]
Could you please share a cleaned version of your config.ini file? I see the `NoSectionError` Error which might point to a missing section in the config.ini

**Comment by rajeefmk** [comment]
Hi Akhater, The NoSectionError is thrown at the first line which is the [Bot] section. 

It looks the config.ini file is not getting uploaded to heroku. I checked the directory of uploaded files. Tried removing it config.ini from .gitignore but still didn't help. Digging more now



![Screenshot 2021-04-10 at 7 10 16 PM](https://user-images.githubusercontent.com/6305121/114271718-79f2e000-9a30-11eb-8431-81e4c42967f0.png)
